src/pages/About.jsx

```diff
@@ -1,5 +1,110 @@
+import { Box, Button, Center, Container, Flex, Grid, GridItem, Heading, Image, Link, LinkBox, LinkOverlay, Text } from '@chakra-ui/react';
+
 const About = () => {
-  return <h2>{'About'}</h2>;
+  return (
+    <Container maxW={{ lg: '100%', md: '80%' }} pt='5rem'>
+      <Grid maxH='10rem' templateRows='1fr' templateColumns={{ base: '1fr', lg: 'repeat(2, 1fr)' }}>
+        <GridItem display={{ base: 'none', lg: 'block' }} colSpan={1} borderLeftRadius='5px'>
+          <Image
+            display={{ base: 'none', lg: 'block' }}
+            src='./src/assets/images/cocktail-pink.jpeg'
+            boxSize='80rem'
+            borderLeftRadius='5px'
+          ></Image>
```

Comment:
```suggestion
          <Image
            display={{ base: 'none', lg: 'block' }}
            src='./src/assets/images/cocktail-pink.jpeg'
            boxSize='80rem'
            borderLeftRadius='5px'
          />
```
Si pas de contenu, tu peux fermer le component direct